### PR TITLE
feat: Adds new MinimalModal component

### PR DESCRIPTION
**Pull request by amir-zahedi**

### CODE REVIEW

File: packages/overdrive/lib/components/MinimalModal/MinimalModal.spec.jsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+	});
+
+	it('should match snapshot with body', () => {
+		expect(
+			render(
+				<MinimalModal isOpen>
+					<p>{testBodyText}</p>
+				</MinimalModal>,
+			).baseElement,
+		).toMatchSnapshot();
```
Comment (smccarthy94) [inline packages/overdrive/lib/components/MinimalModal/MinimalModal.spec.jsx:24]: I can see the new MinimalModal component in the StoryBook from Chromatic but it doesn't seem to show in the Chromatic component list/changes - is that expected?

Comment (amir-zahedi) [reply packages/overdrive/lib/components/MinimalModal/MinimalModal.spec.jsx:24]: Good point @smccarthy94 never really paid much attention to what the change list looks like for new components but like you said the satorybook story has definitely been added so I'd be surprised if future changes would not be picked there

Screen Shot 2021-06-18 at 8 53 24 am

Comment (smccarthy94) [reply packages/overdrive/lib/components/MinimalModal/MinimalModal.spec.jsx:24]: Yeah it definitely looks like future changes will be checked against the snapshot as expected. Was just curious of the behaviour between Chromatic/SB etc in this case :)

Comment (amir-zahedi) [reply packages/overdrive/lib/components/MinimalModal/MinimalModal.spec.jsx:24]: Ok so @smccarthy94 turns out I was wrong and this was explicitly disabled for chromatic as was the Standard modal too. These being modals don't translate to anything meaningful in chromatic unfortunately. When I enabled those I just got the triggers captured and not the actual modals

Comment (amir-zahedi) [reply packages/overdrive/lib/components/MinimalModal/MinimalModal.spec.jsx:24]: Screen Shot 2021-06-18 at 9 25 30 am

**Review by smccarthy94 — APPROVED** [review]

**Pull request merged**